workflow/hpxml_inputs.json

```diff
@@ -2315,6 +2315,16 @@
     "hvac_perf_data_cooling_nom_speed_capacities": "1.00",
     "hvac_perf_data_cooling_max_speed_capacities": "1.00, 1.13"
   },
+  "sample_files/base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-normalized-capacities2.xml": {
+    "parent_hpxml": "sample_files/base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-normalized-capacities.xml",
+    "hvac_perf_data_cooling_outdoor_temperatures": "125.0, 95.0, 82.0",
+    "hvac_perf_data_cooling_min_speed_capacities": "0.267, 0.333, 0.368",
+    "hvac_perf_data_cooling_nom_speed_capacities": ", 1.000",
```

Comment:
Should this syntax be supported?

Reply:
Do you have other cases for taking in empty inputs in a comma separated string? Should we support more forms like "none, 1.0", "null, 1.0", "nil, 1.0" or something?

Reply:
Sounds good, then the current fix should be sufficient. Let me know if you have any questions.

Reply:
I tried this fix in my ResStock branch, and it appears to fix the issue for an ASHP.

But I think this fix needs to be applied to the `set_cooling_systems` method as well?

Reply:
[2ecbc0a](https://github.com/NREL/OpenStudio-HPXML/pull/2045/commits/2ecbc0a53298e28227694be06aa6f996351fcaf1)